MAUI/Barcode/two-dimensional-symbology.md

```diff
@@ -0,0 +1,273 @@
+
+### QR code version
+
+The `CodeVersion` property allows you to set various types of version for QR code from QRVersion enumeration. By default, its value is set as auto.
+
+The QR Code uses version from 1 to 40. Version 1 measures 21 modules x 21 modules, Version 2 measures 25 modules x 25 modules, and so on.
+
+The number of modules increases in steps of 4 modules per side up to Version 40 that measures 177 modules x 177 modules.
+
+{% tabs %}
+
+{% highlight xaml hl_lines="6" %}
+
+    <barcode:SfBarcodeGenerator Value="www.syncfusion.com"
+                                ShowText="True"
+                                HeightRequest="300" 
+                                WidthRequest="300">
+        <barcode:SfBarcodeGenerator.Symbology>
+            <barcode:QRCode CodeVersion="Version09"/>
+        </barcode:SfBarcodeGenerator.Symbology>
+     </barcode:SfBarcodeGenerator>
+
+{% endhighlight %}
+
+{% highlight C# hl_lines="7" %}
+
+    SfBarcodeGenerator barcode = new SfBarcodeGenerator();
+    barcode.HeightRequest = 300;
+    barcode.WidthRequest = 300;
+    barcode.Value = "www.syncfusion.com";
+    barcode.Symbology = new QRCode() 
+    { 
+        CodeVersion = QRCodeVersion.Version09 
+    };
+
+    barcode.ShowText = true;
+    this.Content = barcode;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![.NET MAUI Barcode Generator QR Code Version09](images/two-dimensional-symbology/maui-qr-code-version09.png)
+
+## Data Matrix
+
+`Data Matrix` barcode is a two dimensional barcode that consists of a grid of dark and light dots or blocks forming a square or rectangular symbol. The data encoded in the barcode can either be numbers or alphanumeric. They are widely used in printed media such as labels and letters. You can read it easily with the help of a barcode reader and mobile phones.
+
+{% tabs %}
+
+{% highlight xaml hl_lines="6" %}
+
+    <barcode:SfBarcodeGenerator Value="www.syncfusion.com"
+                                ShowText="True"
+                                HeightRequest="300" 
+                                WidthRequest="300">
+        <barcode:SfBarcodeGenerator.Symbology>
+            <barcode:DataMatrix/>
+        </barcode:SfBarcodeGenerator.Symbology>
+    </barcode:SfBarcodeGenerator>
+
+{% endhighlight %}
+
+{% highlight C# hl_lines="5" %}
+
+    SfBarcodeGenerator barcode = new SfBarcodeGenerator();
+    barcode.HeightRequest = 300;
+    barcode.WidthRequest = 300;
+    barcode.Value = "www.syncfusion.com";
+    barcode.Symbology = new DataMatrix();
+    barcode.ShowText = true;
+    this.Content = barcode;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![.NET MAUI Barcode Generator Data Matrix Symbology](images/two-dimensional-symbology/maui-data-matrix.png)
+
+### Data matrix format
+
+Length - Data Matrix can store up to 2335 alphanumeric characters or 3116 numbers from the ASCII range.
+
+Type - Data Matrix supports the following data types,
```

Comment:
Data Matrix supports the following data types:

Reply:
Changes updated.